src/index.html

```diff
@@ -7,6 +7,330 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/src/styles/main.scss">
+    <title>Moyo header</title>
+  </head>
+  <body>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#menu" class="logo__link">
+        <img
+          src="images/logo.png"
+          alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#APPLE" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link ">SMARTPHONES</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">GADGETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">TABLETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo"class="nav__link">PHOTO</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <main class="main">
+      <article class="card" data-qa="card">
+        <div class="card__img-container">
```

Comment:
This div is redundant, so you need to remove it everywhere
```suggestion
```

Reply:
Agreed with previous comment